oak/proto/storage.proto

```diff
@@ -50,29 +50,35 @@ message DeleteResponse {
 }
 
 message BeginRequest {
-  bytes data_store_id = 1;
+  bytes storage_id = 1;
 }
 
 message BeginResponse {
   bytes transaction_id = 1;
 }
 
 message CommitRequest {
-  bytes transaction_id = 1;
+  bytes storage_id = 1;
+  bytes transaction_id = 2;
 }
 
 message CommitResponse {
 }
 
 message RollbackRequest {
-  bytes transaction_id = 1;
+  bytes storage_id = 1;
+  bytes transaction_id = 2;
 }
 
 message RollbackResponse {
 }
 
 // Wrapper message used to send requests via the StorageChannel interface.
 message StorageOperationRequest {
+  // The name of the storage on which to perform the operation.
+  // The StorageChannel generates a name-based (version 5) UUID from this value.
```

Comment:
Not sure I follow this, is the UUID deterministically generated from the name? What purpose does it serve?